### PR TITLE
[ui][new] adding basic concur connectivity oauth test

### PR DESCRIPTION
**Pull request by mcada**
//test: @oauth

Note that this will only work after PR with credentials update is merged.

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by avano** [comment]
retest this please

File: ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java [hunk 1]
```diff
@@ -616,6 +616,22 @@ private void fillAndValidateGmail() {
...
+            $(By.id("userid")).shouldBe(visible).sendKeys(account.get().getProperty("userId"));
+            $(By.xpath("//*[@type='submit']")).shouldBe(visible).click();
+            TestUtils.sleepForJenkinsDelayIfHigher(3);
+            $(By.id("password")).shouldBe(visible).sendKeys(account.get().getProperty("password"));
+            $(By.xpath("//*[@type='submit']")).shouldBe(visible).click();
+            //give it time to log in
+            TestUtils.sleepForJenkinsDelayIfHigher(4);
+
+        } else {
+            Assert.fail("Credentials for QE Concur were not found.");
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: static import?

File: ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java [hunk 2]
```diff
@@ -616,6 +616,22 @@ private void fillAndValidateGmail() {
...
+
+        if (account.isPresent()) {
+
+            $(By.id("userid")).shouldBe(visible).sendKeys(account.get().getProperty("userId"));
+            $(By.xpath("//*[@type='submit']")).shouldBe(visible).click();
+            TestUtils.sleepForJenkinsDelayIfHigher(3);
+            $(By.id("password")).shouldBe(visible).sendKeys(account.get().getProperty("password"));
+            $(By.xpath("//*[@type='submit']")).shouldBe(visible).click();
+            //give it time to log in
+            TestUtils.sleepForJenkinsDelayIfHigher(4);
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/steps/CommonSteps.java]: isn't there any way how to change this from thread.sleep to some condition what is visible when the user is logged in?

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
@avano this needs to be tested against newer syndesis version, the issue why it failed is already fixed. Also there is another issue with Twitter OAuth, but concur works well on master.

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Pull request merged**